corehq/pillows/case_search.py

```diff
@@ -98,22 +98,21 @@ def _get_case_properties(doc_dict):
     return base_case_properties + dynamic_mapping
 
 
-class CaseSearchPillowProcessor(ElasticProcessor):
-
-    def process_change(self, change):
-        assert isinstance(change, Change)
-        if self.change_filter_fn and self.change_filter_fn(change):
```

Comment:
This is handled in the super class `process_changes_chunk` https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/pillowtop/processors/elastic.py#L135-L139